package.json

```diff
@@ -167,4 +169,4 @@
   "overrides": {
     "cookie": "^0.7.2"
   }
-}
+}
```

Comment:
Missing newline at end of file?

Reply:
Ugh, will need to take a look at my vscode settings - I thought it was prettier that removes it 😬